pkgs/development/python-modules/napari-console/default.nix

```diff
@@ -36,9 +36,6 @@ buildPythonPackage rec {
     qtconsole
```

Comment:
Could you please remove `imageio` and `napri-plugin-engine`, and add `qtpy`?
https://github.com/napari/napari-console/blob/e2a8d7120a179c26295431a5c40820dcda6bb8f0/setup.cfg#L29-L33